lib/Zonemaster/Backend/Config.pm

```diff
@@ -269,15 +272,14 @@ sub parse {
         }
     }
 
-    $obj->{_public_profiles} = {
-        default => undef,
-    };
     for my $name ( $ini->Parameters( 'PUBLIC PROFILES' ) ) {
-        $obj->{_public_profiles}{lc $name} = $get_and_clear->( 'PUBLIC PROFILES', $name );
+        my $path = $get_and_clear->( 'PUBLIC PROFILES', $name );
```

Comment:
Will this remove duplicates? On purpose?

Reply:
It's Config::IniFiles that removes duplicates. I can't say if it's on purpose but I'd prefer if it didn't.